extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToBigDecimalPage.java

```diff
@@ -20,16 +21,15 @@
 
 public class ToBigDecimalPage<ATTR extends Any> implements ToPage<ATTR, BigDecimal[]> {
```

Comment:
I'm curious if we should delete `ToBigDecimalPage` and `ToBigIntegerPage` in order to just use `ToObjectPage`.